client/src/components/CommonComponents/SignIn/index.jsx

```diff
@@ -4,6 +4,7 @@ import 'firebase/auth';
 import 'firebase/firestore';
 import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
 import uiConfig from './uiConfig';
+import styles from './style'; // This uses CSS modules.
```

Comment:
You imported style but you didn`t use/call it, why?

Reply:
yes cause i wanna try to find a way to change the Google sign-in design according to our design